src/utils/variableType.js

```diff
@@ -7,10 +8,28 @@ import {isFunction} from './functionType';
  * @return {Boolean}
  */
 export function isUpdate(node) {
-  return node.type === 'UpdateExpression' && node.argument.type === 'Identifier';
+  return node.type === 'UpdateExpression';
```

Comment:
This looks like a change that should break `let` transform which relies that `isUpdate()` is true when updating only a variable. The tests don't break and it happens to work fine without this, but this might be accident of current implementation.

The functions in this file are specifically design to deal only with variables, better to not change it. For simple function like this one, the comparison can just be written inline.